core/src/create.jl

```diff
@@ -153,7 +154,7 @@ function parse_static_and_time(
                     )
                     if !is_valid
                         errors = true
-                        @error "A $parameter_name time series for $nodetype node #$node_id has repeated times, this can not be interpolated."
+                        @error "A $parameter_name time series for $nodetype node $node_name (#$node_id) has repeated times, this can not be interpolated."
```

Comment:
I expect that node names will often be empty strings. Then this gives a double space `node  (#2)`. How about using `repr(node_name)` such that the quotes are written. That reads nicer for longer names with spaces as well: `node "Lake IJssel" (#2)` / `node "" (#2)`.